src/braket/aws/aws_session.py

```diff
@@ -202,17 +232,14 @@ def create_job(self, **boto3_kwargs) -> str:
         """
         Create a quantum job.
 
-        Args:
-            **boto3_kwargs: Keyword arguments for the Amazon Braket `CreateJob` operation.
```

Comment:
We're inconsistent in how/when we specify args and kwargs. We also don't provide any information in the documentation other than the obvious. I'm ambivalent about removing them, but by it's nature args/kwargs don't have specific meanings (most of the time). So I opted to remove them all. 

The other option would be to always require documentation whenever we have args/kwargs in the signature.

Reply:
OK, I added these back in, since I'm ambivalent about it. We can decide on this later. 

We still have a handful of errors to look at after this PR is done.